new_tests/test_mpc80coltoxml.py

```diff
@@ -0,0 +1,41 @@
+'''
+Test mpc80coltoxml converting a few obs80col format to xml
+with and without the header
+'''
+
+#Import global
+import pytest
+import os
+import subprocess
+
+#Testing file without the header - this test should pass
+def test_mpc80coltoxml_mpcfile():
+    obsfile = "input/K20Q04A_test.obs"
+    xmlfile = "output/K20Q04A_test.xml"
+    if os.path.exists(xmlfile):
+        os.remove(xmlfile)
+    subprocess.run("python3 ../Python/bin/mpc80coltoxml.py "+obsfile+" "+xmlfile,shell=True)
+    assert(os.path.exists(xmlfile) and os.stat(xmlfile).st_size != 0)
+    print('******* Test 1 should pass ********\n')
```

Comment:
On our call, we talked about adding tests that test the `import` procedure as well as the executable procedure. Just a note that the way things are set up, the equivalent to
```
python3 ../Python/bin/mpc80coltoxml.py input output
```
as an import is
```
import mpc80coltoxml
mpc80coltoxml.mpc80coltoxml(["", "input", "output"])
```
where the function call requires a leading value (`""`) be passed in the argument list, since `sys.argv[0]` is the program name. I think that's fine for now, because it meets the goals/requirements of the PR, but this is ultimately undesirable code behavior and user experience.

Reply:
@stevenstetzler yes, you are right. I did not want to change the way in which people are currently calling the script, that's why I kept it that way. But I agree with you that is not the best thing to do.